JRuby.md

```diff
@@ -0,0 +1,117 @@
+## JRuby
```

Comment:
This file should not live at the top-level. If anything it should be in `docs/`

Reply:
Fixed